mllib/src/main/scala/org/apache/spark/mllib/evaluation/MulticlassMetrics.scala

```diff
@@ -153,7 +153,7 @@ class MulticlassMetrics @Since("1.1.0") (predictionAndLabels: RDD[(Double, Doubl
   lazy val recall: Double = accuracy
 
   /**
-   * Returns f-measure
+   * Returns micro averaged f1-measure
```

Comment:
There's more to it than just writing the name. It's accurate but not widely known, so this and the docs above should at least include a link and minimal explanation of what it means.